src/components/Border/Border.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import './Border.css';
+
+function Border() {
+    return(
+        <div className='container-fluid' id='border-container'>
```

Comment:
```suggestion
        <div className='container-fluid border-container'>
```

avoid using id to style components